model-optimizer/extensions/ops/split.py

```diff
@@ -44,7 +44,8 @@ def infer(node):
 
         axis = node.in_port(1).data.get_value() if op == 'VariadicSplit' else node.soft_get('axis', None)
         assert axis is not None, '{} `axis` is unknown for node {}'.format(op, name)
-        assert axis.ndim == 0, '{} `axis` should be scalar, but it`s not for node {}'.format(op, name)
+        assert axis.ndim == 0 or len(axis) == 1 and axis.ndim == 1, '{} `axis` should be scalar or tensor with shape ' \
+                                                                    '[1], but it`s not for node {}'.format(op, name)
```

Comment:
Add unit test to test your changes. I have unchecked "unit tests" item in the PR description.

Reply:
Unit tests were added.